lib/modules/versioning/composer/index.ts

```diff
@@ -44,7 +44,10 @@ function padZeroes(input: string): string {
 
 function convertStabilityModifier(input: string): string {
   // Handle stability modifiers.
-  const versionParts = input.split('@');
+  const versionParts = input
+    // 1.0beta2 to 1.0@beta2
+    .replace(regEx(/(\d+(?:\.\d+)*)(beta|alpha|rc)/gi), '$1@$2')
+    .split('@');
```

Comment:
🤔 shouldn't it be a dash?
```suggestion
    .split('-');
```

Reply:
It's already splitting by `@` because that's the default way to do it in composer. This coerces all ways to eventually be a `-` by the end of the method.